{{cookiecutter.project_name}}/docs/ml-developer-guide.md

```diff
@@ -23,18 +23,18 @@ that are targeted at MLflow Recipes users. Then, when you're ready
 to productionize your ML project, ask your ops team to set up CI/CD and deploy
 production jobs per the [MLOps setup guide](./mlops-setup.md).
 
-### Configure your ML pipeline
+### Configure your ML recipe
```

Comment:
I think we should keep the "pipeline" here.
From what I saw, we only use ML pipeline or MLflow recipe.